src/plots/plots.js

```diff
@@ -2886,13 +2885,25 @@ function sortAxisCategoriesByValue(axList, gd) {
         }
     }
 
+    var aggFn = {
+        'min': function(values) {return Lib.aggNums(Math.min, null, values);},
+        'max': function(values) {return Lib.aggNums(Math.max, null, values);},
+        'sum': function(values) {return Lib.aggNums(function(a, b) { return a + b;}, null, values);},
+        'value': function(values) {return Lib.aggNums(function(a, b) { return a + b;}, null, values);},
```

Comment:
Hmm. Written in this way, I'm thinking maybe _total_ (as in `categoryorder: 'total ascending'`) would make more sense. Thoughts?

Reply:
I agree that `value` is not a good name. I think the vision was to have `value` switch the ordering mode similar to what `array` does (with `array`, we look for the order in `categoryarray`). I think we should reserve `value` for this purpose. I will rename the current `value` to `total`.

Reply:
Done in 9aaa10b59a62cd4595f665ec2860839384d4e047